docs/modules/ROOT/pages/how-tos/metadata/sboms.adoc

```diff
@@ -1 +1,152 @@
+
+== Viewing an SBOM in the web UI
+
+.Prerequisites
+
+* You must have an application that {ProductName} has successfully built and deployed.
+
+.Procedure
+
+In the console, complete the following steps to download the SBOM for a component:
+
+. Navigate to the *Activity* > *Pipeline runs* tab.
+
+. For the component whose SBOM you want to view, select its most recent pipeline run.
+
+. On the *Pipeline run details* page, select *View SBOM*. 
+.. You can use your web browser to immediately search the SBOM for terms that indicate vulnerabilities in your software supply chain. For example, try searching for "log4j".
+.. You can select *Download* to download the SBOM, or *Expand* to view it full-screen.
+
+[NOTE]
+====
+This SBOM is unsigned. You can still use it for standard business processes, like providing an SBOM to your customers. However, do not use this unsigned SBOM as a legal document. 
+====
+
+== Downloading an SBOM in the CLI
+
+.Prerequisites
+
+* Install the link:https://docs.sigstore.dev/cosign/installation/[Cosign] CLI tool.
+
+* Install the link:https://stedolan.github.io/jq/download/[jq] CLI tool.
+
+* xref:/getting-started/cli.adoc[Login] to {ProductName}.
+
+.Procedure
+
+In the CLI, complete the following steps to download the SBOM for a component:
+
+. List your components.
++
+[source]
+----
+$ oc get components
+----
++
+.Example output
++
+[source]
+----
+NAME                               AGE   STATUS   REASON   TYPE
+devfile-sample-go-basic-8wqt       8m54s True     OK       Updated
+devfile-sample-python-basic-ikch   20d   True     OK       Updated
+----
+
+. Choose which component's SBOM you want to download. Then use `oc get` and the jq CLI tool to get the component image path.
+
++
+[source]
+----
+$ oc get component <component name> -ojson | jq '.status.containerImage'
+----
+
++
+.Example
+
++
+[source]
+----
+$ oc get component devfile-sample-python-basic-ikch -ojson | jq '.status.containerImage'
+    "quay.io/redhat-appstudio/user-workload@sha256:<output omitted>"
+----
+
+. Use Cosign to download the SBOM. From the output of the last command, pass the image path as an argument into Cosign's `download sbom` command. Be sure to delete any quotation marks around the image path.
+
++
+.Example
++
+[source]
+----
+$ cosign download sbom quay.io/redhat-appstudio/user-workload@sha256:<output omitted>
```

Comment:
Maybe instead of `<output omitted>`, you can put `<sha-digest>` here, above, and below? It feels odd to have output labeled in the input.